libs/utils/src/lib/fetch-github-count.ts

```diff
@@ -1,4 +1,4 @@
-export const getGithubCount = async () => {
+export const fetchGithubCount = async () => {
```

Comment:
This one should return the same `{ number, formattedResult }` shape as the others.

Reply:
You're right. I corrected that.